docs/technical_reference/flowsheets/mvc.rst

```diff
@@ -0,0 +1,125 @@
+    * `Pressure Changer <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/pressure_changer.html>`_
+    * `Evaporator <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/mvc.html>`_
+    * `Compressor <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/mvc.html>`_
+    * `Condenser <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/mvc.html>`_
+
+Documentation for each of the IDAES unit models can be found below.
+    * `Feed <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/feed.html>`_
+    * `Heat Exchanger <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/heat_exchanger.html>`_
+    * `Separator <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/separator.html>`_
+    * `Product <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/product.html>`_
+    * `Mixer <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/mixer.html>`_
+    * `Translator <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/translator.html>`_
+
+Documentation for each of the property models can be found below.
+    * `Water <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/water.html>`_
+    * `Seawater <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/seawater.html>`_
+
+Documentation for the costing relationships can be found below.
+    * `WaterTAP Costing Package <https://watertap.readthedocs.io/en/latest/technical_reference/costing/watertap_costing.html>`_
+
+The objective function is to minimize the levelized cost of water, which can be represented by the following equation
+where :math:`Q` represents volumetric flow, :math:`f_{crf}` represents capital recovery factor
+:math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and
+:math:`f_{util}` represents the utilization factor:
+
+    .. math::
+
+        LCOW_{Q} = \frac{f_{crf}   C_{cap,tot} + C_{op,tot}}{f_{util} Q}
+
+Degrees of Freedom
+------------------
+The following variables are specified for the MVC flowsheet based on the default settings:
+    * feed water conditions (mass flow, mass fractions, temperature, and pressure)
+    * feed pump efficiency and pressure change (ΔP)
+    * distillate HEX heat transfer coefficient, cold-side ΔP, and hot-side ΔP
+    * brine HEX heat transfer coefficient, cold-side ΔP, and hot-side ΔP
+    * evaporator overall heat transfer coefficient
+    * compressor efficiency
+    * brine pump efficiency and ΔP
+    * distillate pump efficiency and ΔP
+    * translator block outlet TDS concentration
+
+Flowsheet Specifications
```

Comment:
The table needs to be updated based on the modification to the DOF list (i.e., removing the pressure ratio, etc.). Alternately, lower and upper bounds on pressure ratio and evaporator temperature could be added.

Reply:
Good catch - thanks!